packages/compass-schema-validation/src/modules/is-loaded.js

```diff
@@ -0,0 +1,21 @@
+export const IS_LOADED_CHANGED = 'validation/namespace/IS_LOADED_CHANGED';
+
+export const INITIAL_STATE = {
+  collectionReadOnly: false,
+  hadronReadOnly: false,
+  writeStateStoreReadOnly: false,
+  oldServerReadOnly: false
```

Comment:
I think initial state should just be a `false` boolean? Maybe copy paste cleanup?

Reply:
Yes I clearly copy/pasted things from two different places 🤦

Reply:
Pushed a change and tests.

Reply:
Aargh now something is wrong when I test manually. It shows the zero state momentarily again. Investigating..

Reply:
OK I think I just dispatched things in the wrong order. Making sure I change the is-loaded state last now.